source/widgets/settings_window/blender_builds_tab.py

```diff
@@ -1,3 +1,4 @@
+from requests import get
```

Comment:
```suggestion
```
While `requests` is a secondary dependency, it's not used in the code and only exists if you install the repo in dev mode